Jenkinsfile

```diff
@@ -0,0 +1,112 @@
+//"Jenkins Pipeline is a suite of plugins which supports implementing and integrating continuous delivery pipelines into Jenkins. Pipeline provides an extensible set of tools for modeling delivery pipelines "as code" via the Pipeline DSL."
+//More information can be found on the Jenkins Documentation page https://jenkins.io/doc/
+
+@Library('github.com/connexta/cx-pipeline-library@master') _
+@Library('github.com/connexta/github-utils-shared-library@master') __
+
+pipeline {
+    agent {
+        node {
+            label 'linux-large-ddf'
```

Comment:
I would keep this large (based on what I know), but curious if has to be large

Reply:
it worked about as fast with linux-medium and linux-small so I changed to small